packages/webpack-cli/lib/groups/ZeroConfigGroup.js

```diff
@@ -3,6 +3,13 @@ const { logger } = require('@webpack-cli/logger');
 
 const PRODUCTION = 'production';
 const DEVELOPMENT = 'development';
+const NONE = 'none';
+let isNoMode = false;
+process.argv.forEach((value) => {
```

Comment:
That's not how you should retrieve the information.
Inside the constructor of this file, you have `options` and after the `super` call you will have access to `this.args` too.

Please retrieve the information from `options` and `this.args`